packages/containers/Team/Team.js

```diff
@@ -273,6 +273,7 @@ export const TeamContainerComponent = ({ currentUser, team, members }) => {
       {modalVisibilities[MODAL.NEW_TEAM] && (
         <TeamModal
           teamId={team.id}
+          teams={Object.values(teams)}
```

Comment:
You may use teamListSelector. This is array view of teams and made with Object.values() like you did.